lib/tsc-watch.js

```diff
@@ -50,12 +51,15 @@ tscProcess.stdout.on('data', buffer => {
   const lines = manipulate(buffer);
   print(noColors, noClear, lines);
   const state = detectState(lines);
+  const newCompilation = state.newCompilation;
   const compilationError = state.compilationError;
   const compilationComplete = state.compilationComplete;
 
+  compilationErrorSinceStart = (!newCompilation && compilationErrorSinceStart) || compilationError;
```

Comment:
Why would you need to test for `newCompilation`? `compilationError` is not enough?
I don't really understand your intents here. please explain